docker-bake.hcl

```diff
@@ -65,6 +66,20 @@ target "flow-worker" {
   ]
 }
 
+target "flow-maintenance" {
```

Comment:
is this something that would go into a critical node pool?

Reply:
We can just yeet it on low cpu one